core/Flist.cva6

```diff
@@ -32,6 +32,7 @@ ${CVA6_REPO_DIR}/vendor/pulp-platform/fpga-support/rtl/AsyncThreePortRam.sv
 
 +incdir+${CVA6_REPO_DIR}/vendor/pulp-platform/common_cells/include/
 +incdir+${CVA6_REPO_DIR}/vendor/pulp-platform/common_cells/src/
++incdir+${CVA6_REPO_DIR}/vendor/pulp-platform/axi/include/
```

Comment:
I took time some month ago to remove this package dependency. Tho objective was to pass axi information through the cva6 input parameters. It would be nice to remove this package use.

Reply:
This does not compile any package. This give access to AXI header files that can be included such as 'axi/typedef.svh'.